docs/source/data_readers.rst

```diff
@@ -50,9 +50,11 @@ be specified using the options dict parameter.
 Possible `options`:
 
 * delimiter - Must be a string, for example `"delimiter": ","`
-* data_format - must be a string, possible choices: "dataframe", "records"
-* selected_columns - columns being selected from the entire dataset, must be a 
+* data_format - Must be a string, possible choices: "dataframe", "records"
+* selected_columns - Columns being selected from the entire dataset, must be a 
   list `["column 1", "ssn"]`
+* sample_nrows - Reservoir sampling to sample `n` rows out of a total of `M` rows.
+  Specified for how many rows to sample, default None.
```

Comment:
this is new ... to reflect https://github.com/capitalone/DataProfiler/blob/dev/dataprofiler/data_readers/csv_data.py#L90